Source/init.cpp

```diff
@@ -157,6 +157,7 @@ void init_create_window(int nCmdShow)
 	wcex.hIconSm = (HICON)LoadImage(ghInst, MAKEINTRESOURCE(IDI_ICON1), IMAGE_ICON, 16, 16, LR_DEFAULTCOLOR);
 	if (!RegisterClassEx(&wcex))
 		app_fatal("Unable to register window class");
+#endif

```

Comment:
Wait is this needed?

Reply:
Probably not, but cannot test atm. Will test later.

Reply:
Not needed, you were right.

Fixed in https://github.com/diasurgical/devilutionX/pull/201/commits/00ee02c4d39f9bdf338ff9b305bf13666a0afb83